composer.json

```diff
@@ -33,7 +33,8 @@
     "znk3r/html_quickform": "4.0.2",
     "friendsofpear/pear_exception": "0.0.*",
     "znk3r/html_common": "*",
-    "pear/pear": "^1.10"
+    "pear/pear": "^1.10",
+    "doctrine/dbal": "^2.5"
```

Comment:
pourquoi ajouter doctrine/dbal ça fait deux façons de requêter. ting devrais suffir, non ?

Reply:
Ting est trop limité, j'ai cherché un bon moment pour faire un truc tout simple (passer un tableau de valeurs à binder pour faire un `IN(:values)`) mais ce n'est pas géré... Doctrine DBAL est beaucoup plus puissant et tout aussi léger, beaucoup mieux maintenu et tout le monde l'utilise donc pas besoin d'apprendre un nouvel outil, ce qui est un gros avantage pour motiver davantage de personnes à contribuer au site.